plugins/analysis-nori/src/test/java/org/elasticsearch/plugin/analysis/nori/NoriAnalysisTests.java

```diff
@@ -121,7 +121,7 @@ public void testNoriAnalyzerInvalidUserDictOption() throws Exception {
     public void testNoriAnalyzerDuplicateUserDictRule() throws Exception {
         Settings settings = Settings.builder()
             .put("index.analysis.analyzer.my_analyzer.type", "nori")
-            .put(IndexMetadata.SETTING_VERSION_CREATED, IndexVersions.ES_VERSION_8_13)
+            .put(IndexMetadata.SETTING_VERSION_CREATED, IndexVersions.ES_VERSION_8_13) // HERE
```

Comment:
This will be removed. It is highlighted here to show the usage of ES_VERSION_8_13, since I'm proposing to reuse that name with a new id.